src/test/resources/com/conveyal/datatools/gtfs/fake-agency-expire-in-2099-with-unused-route/calendar.txt

```diff
@@ -0,0 +1,3 @@
+service_id,monday,tuesday,wednesday,thursday,friday,saturday,sunday,start_date,end_date
+common_id,1,1,1,1,1,1,1,20170915,20990917
+both_id,1,0,1,0,1,0,0,20170915,20990917
```

Comment:
It looks like this was copied from one of the feed merging test cases which I guess is fine, but it might have been simpler to copy from the fake-agency-with-only-calendar folder since it has less data which could help reduce the amount of nuances with schedules while testing.

Reply:
Updated.